lib/targets.go

```diff
@@ -31,10 +31,13 @@ type Target struct {
 // Request creates an *http.Request out of Target and returns it along with an
 // error in case of failure.
 func (t *Target) Request() (*http.Request, error) {
-	req, err := http.NewRequest(t.Method, t.URL, bytes.NewReader(t.Body))
+	req, err := http.NewRequest(t.Method, t.URL, nil)
 	if err != nil {
 		return nil, err
 	}
+	if req.Method != http.MethodGet {
```

Comment:
This is a backwards incompatible change. I'll push a follow-up commit that sets no body if and only if t.Body is empty or nil, not if the request method is GET.